src/bot/commands/games/akinator.ts

```diff
@@ -0,0 +1,129 @@
+import { Command, Input } from '@api';
+import { Message } from 'discord.js';
+import { Akinator } from '@libraries/akinator';
+
+let images: string[] = [
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/defi.png?v90',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/inspiration_legere.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/etonnement.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/serein.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/vrai_decouragement.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/inspiration_forte.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/confiant.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/mobile.png',
+    'https://en.akinator.com/bundles/elokencesite/images/akitudes_670x1096/surprise.png',
+]
+
+export class AkinatorGame extends Command {
+    constructor() {
+        super({
+            name: 'akinator',
+            description: 'Starts the akinator game.',
+            aliases: ['aki']
+        });
+    }
+
+    async execute(input: Input) {
+        let akinator = new Akinator('en');
+
+        // Start the session
+        let step = await akinator.start();
+        this.getLogger().debug(`Started akinator game with session ${step.session.sessionId} and signature ${step.session.sessionSignature}.`);
+
+        // Send the initial message
+        let message = await input.channel.send({
+            embed: {
+                color: 0x499df5,
+                thumbnail: { url: _.sample(images) },
+                fields: [{
+                    name: `${input.member.displayName}, Question ${step.number}`,
+                    value: `**${step.question}**\nyes (**y**) / no (**n**) / idk (**i**) / probably (**p**) / probably not (**pn**)\nback (**b**)`,
+                }]
+            }
+        }) as Message;
+
+        // Main game loop
+        while (true) {
+            this.getLogger().debug(`Session ${step.session.sessionId}: ${step.question} (${step.certainty}%)`);
+
+            // Get the user's response as a number
+            let reaction = await this.getResponse(input);
+            this.getLogger().debug(`Session ${step.session.sessionId}: Answered ${reaction}`);
+
+            // Go back to the previous step
+            if (reaction == 5) {
+                if (step.number == 1) break;
+                step = await akinator.previous();
+            }
+
+            // Or continue to the next step
+            else {
+                step = await akinator.next(reaction);
+            }
+
+            // If we are 85%+ certain on who the character is, display it
+            if (step.certainty >= 85) {
+                message.deleteAfter(10000);
```

Comment:
You could move the time values for deleting a message to a constant variable at the top of the file, just so they're easier to swap out.

Reply:
I'll do that, thanks for the reminder, I have it set that way in some of the other commands I did